### PR TITLE
Fix typos and style in faq/rsh.inc

### PR DESCRIPTION
**Pull request by apjanke**

### CODE REVIEW

File: faq/rsh.inc [hunk 1]
```diff
@@ -141,13 +141,13 @@ permissions:
 
 The phrase \"at least\" in the above paragraph means the following:
 <ul>
-<li> The files need to be readable by you</li>
-<li> The files should _only_ be writable by you</li>
-<li> The files should not be executable</li>
+<li> The files need to be readable by you.</li>
```
Comment (apjanke) [inline faq/rsh.inc]: These list items should end with periods, because they are complete sentences.

File: faq/rsh.inc [hunk 2]
```diff
@@ -104,7 +103,8 @@ Accept the default value for the file in which to store the key
 may choose to not enter a passphrase and therefore obviate the need
 for using the [ssh-agent].  However, this <strong>greatly</strong>
 weakens the authentication that is possible, because your secret key
-is [potentially] vulnerable to compromise because it is unencrypted.
+is <strong>potentially</strong>strong> vulnerable to compromise
```
Comment (jsquyres) [inline faq/rsh.inc]: Typo here: extra `strong>`.

File: faq/rsh.inc [hunk 3]
```diff
@@ -120,11 +120,11 @@ shell$ cp id_rsa.pub authorized_keys
...
 In order for RSA authentication to work, you need to have the
 [\$HOME/.ssh] directory in your home directory on all the machines you
-are running Open MPI.  If your home directory is on a common
+are running Open MPI on.  If your home directory is on a common
 filesystem, this may be already taken care of.  If not, you will need to
 copy the [\$HOME/.ssh] directory to your home directory on all Open
-MPI nodes (be sure to do this in a secure manner &mdash; perhaps using the
-[scp] command &mdash; particularly if your secret key is not encrypted).
+MPI nodes. (Be sure to do this in a secure manner &mdash; perhaps using the
+[scp] command &mdash; particularly if your secret key is not encrypted.ß)
```
Comment (jsquyres) [inline faq/rsh.inc]: Typo here: extra `ß`

**Comment by apjanke** [comment]
Typos fixed!

**Pull request merged**